docs/API.md

```diff
@@ -1,5 +1,107 @@
 # API
 
+## `createMaterialPalette()`
```

Comment:
I would say to divide this doc into section as I did in `withMaterialPalette` to:
- example of usage
- description
- syntax
- flow types
- arguments
- more examples (this can be also concatenated with _example of usage_